src/test/java/io/a2a/client/A2AClientTest.java

```diff
@@ -73,37 +71,46 @@ public void tearDown() {
     }
 
     @Test
-    public void testA2AClientSendTask() throws Exception {
+    public void testA2AClientSendMessage() throws Exception {
         this.server.when(
                         request()
                                 .withMethod("POST")
-                                .withPath("/tasks/send")
-                                .withBody(JsonBody.json(SEND_TASK_TEST_REQUEST))
+                                .withPath("/message/send")
+                                .withBody(JsonBody.json(SEND_MESSAGE_TEST_REQUEST, MatchType.STRICT))
 
                 )
                 .respond(
                         response()
                                 .withStatusCode(200)
-                                .withBody(SEND_TASK_TEST_RESPONSE)
+                                .withBody(SEND_MESSAGE_TEST_RESPONSE)
                 );
 
         A2AClient client = new A2AClient("http://localhost:4001");
-        Message message = toUserMessage("tell me a joke");
-        TaskSendParams params = new TaskSendParams.Builder()
-                .id("task-1234")
-                .sessionId("session-1234")
+        Message message = new Message.Builder()
+                .role(Message.Role.USER)
+                .parts(Collections.singletonList(new TextPart("tell me a joke")))
+                .contextId("context-1234")
+                .messageId("message-1234")
+                .build();
```

Comment:
This is a general comment, not impacting this PR...but I'm wondering who ensure contextId and messageID are unique

Reply:
Good question. In general, a messageId is determined by the creator of the message (client or server). If it's not explicitly specified, it's randomly generated. The contextId will be created by the server. I'll create an issue to track this.